apps/prerender/src/components/Profile.tsx

```diff
@@ -17,7 +16,7 @@ interface ProfileProps {
   publications: Publication[];
 }
 
-const Profile: FC<ProfileProps> = ({ profile, publications }) => {
+const ProfileComponent: FC<ProfileProps> = ({ profile, publications }) => {
```

Comment:
Why we need to rename this? now type will not be a conflict because their import is different IMO

Reply:
Hey, nah, it's not about the import.

Profile component uses ProfileProps, which in turn uses Profile type. Hence the clash. https://github.com/lensterxyz/lenster/blob/25d6cf53d831021c1f121a53764e051899d2022b/apps/prerender/src/components/Profile.tsx#L15